.deploystack/clean

```diff
@@ -0,0 +1,85 @@
+# DON'T REMOVE FROM test script.
```

Comment:
**Suggestion:**
Since most Online Boutique maintainers are unfamiliar with DeployStack, could you please add a comment at the top of this file explaining its purpose (e.g., when/where does it get used)?

Reply:
Added documentation

Reply:
Thanks for adding these comments!